CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+- Add: `advanced_event_selector` in examples
+- Change: Use upgraded S3 module for external bucket
+- Allow overwriting the default KMS Policies
+
+## [1.1.0] - 2022-05-25
+### Added
+- Added the `.github/workflow` folder (not supposed to run gitcommit)
```

Comment:
Add the KMS support and update here, also look at the date, it should be advanced :)